packages/core/src/libraries/logto-config.ts

```diff
@@ -97,5 +101,28 @@ export const createLogtoConfigLibrary = ({
     return z.object({ value: jwtCustomizerConfigGuard[key] }).parse(rows[0]);
   };
 
-  return { getOidcConfigs, getCloudConnectionData, upsertJwtCustomizer, getJwtCustomizer };
+  /**
+   * We only guard the type of `key` field here.
+   *
+   * The guard of `value` field is handle on:
+   * 1. API request interface.
+   * 2. Manually run zod guard after merging the `value` and the existing record.
+   */
+  const updateJwtCustomizer = async <T extends LogtoJwtTokenKey>(
+    key: T,
+    value: Record<string, unknown>
```

Comment:
should we guard the value here? e.g. Partial<z.infer<(typeof jwtCustomizerConfigGuard)[T]>